packages/caliper-fabric/lib/ConnectorConfiguration.js

```diff
@@ -86,6 +86,17 @@ class ConnectorConfiguration {
 
         return null;
     }
+    /**
+     * return array of contract definitions for specific channel
+     * @param {string} channelPassed channel wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelPassed) {
+        const channelList = this.adapterConfiguration.channels;
+        const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelPassed);
```

Comment:
As we don't have a validator yet and not sure when a validator would be made available for now lets add some defensive checks here to make sure that there is a channelList, so you can check the example used in getChannelNamesForCreation.